bin/docker/Dockerfile

```diff
@@ -18,9 +19,7 @@ COPY ./metabase.jar /app/
 COPY ./run_metabase.sh /app/
 RUN chmod 755 /app/run_metabase.sh
```

Comment:
Also, not related to this change, but is this line even necessary? It looks like it's already 755 in the repo.